src/quality-level-list.js

```diff
@@ -0,0 +1,135 @@
+import videojs from 'video.js';
+import document from 'global/document';
+
+/**
+ * A list of QualityLevels.
+ *
+ * interface QualityLevelList : EventTarget {
```

Comment:
Is this interface based on something external? Might be worth adding a link if so.